k2/csrc/connect.cu

```diff
@@ -536,18 +491,9 @@ class Connector {
   // of that state as an idx012.
   Ragged<int32_t> incoming_arcs_;
   // With the Dim() the same as num-states, to mark the state (as an idx01) to
-  // be accessible or not
+  // be accessible/coaccessible or not. For each element in this array the first
+  // bit uses to mark accessible and the second bit to mark coaccessible.
   Array1<char> accessible_;
```

Comment:
Shall we give it a better name? It is called accessible but it contains information for both accessible and co-accessible.

Reply:
Using the concatenation of accessible and co-accessible, say `accessible_coaccessible_` ?
I think the current name is acceptable, we have explanation in the document.

Reply:
Great!
Either way OK with me, RE name.

Reply:
Fangjun can merge when ready..